cdmtaskservice/s3/remote.py

```diff
@@ -117,21 +119,26 @@ async def download_presigned_url(
         got_etag = calculate_etag(outputpath, partsize)
         if got_etag == etag:
             return False
-    async with session.get(url, ssl=not insecure_ssl) as resp:
-        if resp.status > 199 and resp.status < 300:  # redirects are handled automatically
-            serv_etag = resp.headers["Etag"].strip('"')
-            if etag and etag != serv_etag:
-                raise FileChangeError(
-                    f"Etag check failed for url {url.split('?')[0]}. "
-                    + f"Server provided {serv_etag}, user provided Etag requirement is {etag}")
-            outputpath.parent.mkdir(exist_ok=True, parents=True)
-            with open(outputpath, "wb") as f:
-                async for chunk in resp.content.iter_chunked(_CHUNK_SIZE_64KB):
-                    f.write(chunk)
-        else:
-            # assume the error output isn't too huge
-            err = await resp.read()
-            raise TransferError(f"GET URL: {url.split('?')[0]} {resp.status}\nError:\n{err}")
+    tout = aiohttp.ClientTimeout(total=timeout_sec)
+    try:
+        async with session.get(url, ssl=not insecure_ssl, timeout=tout) as resp:
+            if resp.status > 199 and resp.status < 300:  # redirects are handled automatically
+                serv_etag = resp.headers["Etag"].strip('"')
```

Comment:
```suggestion
                serv_etag = resp.headers.get("Etag", "").strip('"')
```
Does `Etag` always exist?

Reply:
For 3 yes

Reply:
👍